src/parser/plugins/ResourcePlugin.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
IIUC, you can/should add your name/company, i.e. `Copyright 2018 by NetCentric`, since the entire file is your contribution. @stevengill correct?